ImageProcessor.py

```diff
@@ -0,0 +1,21 @@
+class ImageProcessor:
```

Comment:
Add docstrings to the class and methods: Docstrings provide documentation for your code and make it easier for other developers to understand how to use your class and its methods. You can add a docstring to the class and to each method.